base/utferror.jl

```diff
@@ -0,0 +1,32 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+#=#!
+@brief      Error messages for Unicode / UTF support
```

Comment:
https://github.com/JuliaLang/julia/pull/11551#discussion_r31588648 still applies. Please cut out the doxygen stuff, we don't use it anywhere else (remember utf8proc is a separate project that we just happen to depend on) and this looks like a commented out Julia macro if you're not a doxygen user. Just plaintext comments for now and we should be able to get this merged very quickly.